projects/36/bhxiv/paper.md

```diff
@@ -171,9 +171,9 @@ The yaml contains the instructions to create the postgres container and a tool f
 ```
 docker-compose up
 ```
-Then after logging in to the administrattion tool at http://localhost:8081 it must be created a db called "synthea10" and inside it two schema named "cdm_synthea10" and "native".
+Then after logging in to the administration tool at http://localhost:8081 it must be created a db called "synthea10" and inside it two schema named "cdm_synthea10" and "native".
 
-In the third step we chose the athena site[^4] to donwload the vocabularies. Accepting all the default vocabularies resulted in a zip file of over 4 GB and an error later in the fourth step. Reducing the vocabularies zip file dimension under 3 GB solved the issue.
+In the third step we chose the athena site[^4] to download the vocabularies. Accepting all the default vocabularies resulted in a zip file of over 4 GB and an error later in the fourth step. Reducing the vocabularies zip file dimension under 3 GB solved the issue.
 
```

Comment:
Hi @sasurfer Thanks for your review and PR. I cannot see what is the change(s) you proposed here. Could you clarify? Thanks

Reply:
typo. download->donwload

Reply:
ouch! right :) Thanks!!